gnomad_qc/v4/resources/annotations.py

```diff
@@ -14,19 +14,22 @@
 SUBSETS = SUBSETS["v4"]
 
 
-def _annotations_root(version: str = CURRENT_VERSION, test: bool = False) -> str:
+def _annotations_root(
+    version: str = CURRENT_VERSION, data_type: str = "exomes", test: bool = False
+) -> str:
     """
     Get root path to the variant annotation files.
 
     :param version: Version of annotation path to return.
+    :param data_type: default is "exomes", can also be "genomes"
     :param test: Whether to use a tmp path for analysis of the test VDS instead of the
         full v4 VDS.
```

Comment:
```suggestion
    :param test: Whether to use a tmp path for analysis of the test VDS instead of the
        full v4 VDS.
    :param data_type: Data type of annotation resource. e.g. "exomes" or "genomes". Default is "exomes".
```